Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -808,8 +808,8 @@ YUI.add('ez-platformuiapp', function (Y) {
                     view: "serverSideView",
                     callbacks: ['open', 'checkUser', 'handleSideViews', 'handleMainView']
                 }, {
-                    name: "adminRole",
-                    regex: /\/admin\/(pjax%2Frole.*)/,
+                    name: "adminLanguage",
```

Comment:
see further up, @glye added a second adminRole above which is now the correct one I think given it now needs own service.